Decrease retry timeout for trace logging (#15611)
mlflow/mlflow: docs/docs/tracing/production.mdx

For more details about the product and how to set it up, please refer to the [Lakehouse Monitoring for GenAI](https://docs.databricks.com/aws/en/generative-ai/agent-evaluation/monitoring) documentation.

### Configurations

> When using Lakehouse Monitoring for GenAI, MLflow will log traces in **non-blocking manner** by default, such that the application can continue to serve requests without waiting for the trace to be logged. You can configure this logging behavior by setting the following environment variables:

Reviewer: ```suggestion
When using Lakehouse Monitoring for GenAI, MLflow logs traces asynchronously by default, allowing your application to continue serving requests without waiting for trace logging to complete.
```
